apprentice_teacher.rb

```diff
@@ -26,27 +23,6 @@ def teach_stuff
     response
   end
 
```

Comment:
teach_stuff method is common in both the Teacher and ApprenticeTeacher classes except for a few words. Can you think of a way to extract it into the Parent class?